docker-compose.yaml

```diff
@@ -3,7 +3,7 @@ services:
   xnat:
     build:
       context: .
-    image: ghcr.io/brown-bnc/xnat:1.8.7.1
+    image: ghcr.io/brown-bnc/xnat:1.8.10.0-dqr
```

Comment:
Just a question – this looks like the image which is going to be built from this version of the repo. Is that right? What's the deploy process?

Reply:
That's correct—the tag with which I plan to name the image. This hasn't been part of my process in the past, but I figured it would be helpful to update the docker-compose file to reflect the latest image.  However, this won't affect the deployment process.  Only helpful for someone using docker-compose to build XNAT locally.  Our Kubernetes deployment pulls the image straight from the Github Registry, which is built on PRs and via the publish workflow action.